File: LatPrak2_1 (JAVA)/Hardware.java
class Hardware extends Product{
	//atribut
	private String brand;
	private String model;

	//constructor
	Hardware(){
	}

	//get & set untuk atribut brand
	public void setBrand(String brand){
		this.brand = brand;
	}
		
	public String getBrand(){
		return this.brand;
	}

	//get & set untuk atribut model
	public void setModel(String model){
		this.model = model;
	}
		
	public String getModel(){
		return this.model;
	}
}

File: LatPrak2_2 (C++)/Hardware.cpp
class Hardware : public Product{
	private:
	//atribut dalam class Hardware
		string brand;
		string model;

	public:
		//constructor
		Hardware(){
		}

		//prosedur dan fungsi untuk get & set dari atribut Brand
		void setBrand(string brand){
			this->brand = brand;
		}
		
		string getBrand(){
			return this->brand;
		}

		//prosedur dan fungsi untuk get & set dari atribut model
		void setModel(string model){
			this->model = model;
		}
		
		string getModel(){
			return this->model;
		}
	//destructor
	~Hardware(){
	}
};